lib/src/main/java/me/onebone/toolbar/ToolbarWithFabScaffold.kt

```diff
@@ -0,0 +1,101 @@
+package me.onebone.toolbar
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.SubcomposeLayout
+import androidx.compose.ui.unit.LayoutDirection
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun ToolbarWithFabScaffold(
```

Comment:
May I ask you to mark it as `@ExperimentalToolbarApi`? I'd like to put a few new components which may result in breaking function signature and behaviors.

Reply:
Added in new commit